install.md

```diff
@@ -250,6 +252,8 @@ To delete the remaining resources run:
 kubectl delete pods,serviceaccount,clusterrolebinding --selector kritis.grafeas.io/install --namespace <your namespace>
```

Comment:
Can you break this command apart so that it fits in a browser window?

```
kubectl delete pods,serviceaccount,clusterrolebinding \
  --selector kritis.grafeas.io/install \
  --namespace <your namespace>
```

Reply:
done